test/suites/integration/testdata/amd_driver_input.txt

```diff
@@ -0,0 +1,16 @@
+MIME-Version: 1.0
```

Comment:
Should this be a .sh? Does it make sense to align the other files in this directory too, while we're in here?

Reply:
I can do that